templates/prometheus.yaml

```diff
@@ -90,6 +94,25 @@ spec:
           thanos:
             baseImage: thanosio/thanos
             version: v0.4.0
+          externalLabels:
+            cluster: $(CLUSTER_ID)
+          containers:
+            # note: in order to run this container, mesosphereResources.enableAdditionalPrometheusRBACRules
+            # needs to be set to true
+            - name: get-cluster-id
+              image: mesosphere/kubeaddons-addon-initializer:v0.0.13
+              command: ["/bin/bash", "-c", "addon-initializer prometheus && sleep infinity"]
+              env:
+              - name: "PROMETHEUS_NAMESPACE"
+                value: "kubeaddons"
+              - name: "CLUSTER_ID_CONFIGMAP_NAME"
+                value: "cluster-info-configmap"
+              - name: "KUBESYSTEM_NAMESPACE"
```

Comment:
The kube-system namespace will always be the same. This env variable seems redundant to me

Reply:
it's here as more of a precaution, but can remove. extrasteps defaults it to `metav1.NamespaceSystem` (`kube-system`) which is safe, but if we ever do for some reason need to change it, we can pass it in here to override then.